### DISCUSSION TITLE
Is it have a video on how to download it and run it? I am a total noob

### DISCUSSION THREAD

**Discussion by matt00112233** [Serene-Arc/bulk-downloader-for-reddit#431] [General]
I tried my best to download the program but I am lost and I don't know what to do after (see attachment)
![Screenshot 2021-05-26 151327](https://user-images.githubusercontent.com/36396935/119717979-0288dc80-be35-11eb-9baa-5512590175eb.jpg)

**Comment by aliparlakci** [comment]
why are you not using the exact commands we give? Command is **`python -m bdfr download`**, not ~~`-m bdfr download`~~

**Reply by matt00112233** [reply]
Thanks for the quick response, this is what I get when I use the command python -m bdfr download:

C:\Users\Yoga 14>python -m bdfr download
Usage: python -m bdfr download [OPTIONS] DIRECTORY
Try 'python -m bdfr download --help' for help.

Error: Missing argument 'DIRECTORY'.

C:\Users\Yoga 14>

**Reply by matt00112233** [reply]
Let's say my username on Reddit is John10 and I want to download my 5 latest upvoted posts, what would be the command for that?

**Reply by Fakeaccount12312** [reply]
`python -m bdfr download "" --user John10 --upvoted --limit 5 --authenticate`

Replace "" with another path if you don't want the posts in the same directory.

**Reply by matt00112233** [reply]
Thanks a lot, it works nicely, my mistake was that I was including a space after "--". 
But for some subreddits such as PublicFreakout, it only downloads the audio of the videos instead of downloading the videos with audio

**Reply by Fakeaccount12312** [reply]
Does it show errors when downloading these files?

**Reply by matt00112233** [reply]
No it doesn't show any errors, it downloads the posts correctly. For example when I do:

C:\Users\Yoga 14>python -m bdfr download ./path/to/output --subreddit publicfreakout --sort new --limit 5
[2021-05-26 17:15:46,926 - bdfr.downloader - INFO] - Downloaded submission nlrk6h from PublicFreakout
[2021-05-26 17:15:48,711 - bdfr.downloader - INFO] - Downloaded submission nlrinb from PublicFreakout
[2021-05-26 17:15:50,979 - bdfr.downloader - INFO] - Downloaded submission nlrjfu from PublicFreakout
[2021-05-26 17:15:52,338 - bdfr.downloader - INFO] - Downloaded submission nlrbsw from PublicFreakout
[2021-05-26 17:15:55,943 - bdfr.downloader - INFO] - Downloaded submission nlr1jx from PublicFreakout
[2021-05-26 17:15:55,944 - root - INFO] - Program complete

**Reply by Fakeaccount12312** [reply]
I had the same problem, just found out what I did back then with the help of my browsing history. You need ffmpeg to do so, get it from [here](https://www.ffmpeg.org/) and make it aviable for python, I just put the exe in python/python39/scripts.

**Reply by Fakeaccount12312** [reply]
Reddit seems to have some server problems rn.

**Reply by matt00112233** [reply]
Thanks a lot... Which one do you download it and run the exe ? There seems to be different versions of it

**Reply by Fakeaccount12312** [reply]
Are you on windows? If so, you can take [the same one I used](https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-05-25-12-43/ffmpeg-N-102605-g4c705a2775-win64-gpl.zip).

**Reply by matt00112233** [reply]
Yes I am on Windows. Thanks

**Reply by Fakeaccount12312** [reply]
Did it work?

**Reply by matt00112233** [reply]
Thanks so much... It works now so far
